doc/Data Plane Code Generation/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -1,201 +1,146 @@
+```
+e.g.
+```
+dotnet new dataplane --libraryName DeviceUpdate --swagger https://github.com/dpokluda/azure-rest-api-specs/blob/be397aa65510bd4e8f87da539af2b0025f6f44ca/specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/preview/2020-09-01/deviceupdate.json --securityScopes https://api.adu.microsoft.com/.default --force
 ```
 
-#### 3. Explore the Generated Code
+#### 3. update the solution file if needed
+Run 'dotnet sln' as following to update the projects in the solution file:
+dotnet sln remove src\Azure.<grouop>.<service>.csproj
+dotnet sln add src\Azure.<grouop>.<service>.csproj
+dotnet sln remove tests\Azure.<grouop>.<service>.Tests.csproj
+dotnet sln add tests\Azure.<grouop>.<service>.Tests.csproj
 
-After you've run the `GenerateCode` command above, you should find a `Generated` folder in your `Azure.<group>.<service>` project.
+**Learn more:** to understand more about the Azure SDK repo structure, see [Repo Structure](https://github.com/Azure/azure-sdk/blob/master/docs/policies/repostructure.md) in the `azure-sdk` repo.
 
-If you're interested, you can look through the swagger file to see how the autorest.csharp extension translated the REST API definition into C# code. This could help you later if you decide to make adjustments to your API design in the swagger file directly, so they can be used for all Tier 1 languages.
+### Generate client library
 
-**Learn more**: see the [OpenAPI Swagger Documentation](https://swagger.io/docs/) to learn more about swagger file formats.
+From a PowerShell command prompt, navigate to the directory holding `Azure.<group>.<service>.csproj`. Run the following commands:
+```
+sdk\<your-service-name>\Azure.<group>.<service>\src> dotnet build /t:GenerateCode
+```
 
-#### 4. Save and commit your changes to your branch
+### Export Public API
 
-This will make it easier to see the changes you make when you re-generate the code later.
+If you make public API changes or additions, the `eng\scripts\Export-API.ps1` script has to be run to update public API listings. This generates a file in the library's directory similar to the example found in [sdk\template-dpg\Azure.Template.Generated\api\Azure.Template.Generated.netstandard2.0.cs](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template-dpg/Azure.Template.Generated/api/Azure.Template.Generated.netstandard2.0.cs).
 
-Congratulations, you've generated your first Azure SDK library! Next, we'll discuss how we can add tests, samples and documents.
+Running the script for a project in `sdk\deviceupdate` would look like this: 
+```
+eng\scripts\Export-API.ps1 deviceupdate
+```
 
-## Tests
+## Improve the SDK library package
+You can continuously work on the first-version shippable package created above to refine, such as add Tests, Samples and refine the Readme.md and Changlog.
+### Tests ###
```

Comment:
Why is `###` needed after the Tests?

Reply:
Let's please address Shivangi's comment.

Reply:
updated